src/API/Endpoints/Reports/Endpoint.php

```diff
@@ -410,13 +410,70 @@ public function getPayments( $startStr, $endStr, $orderBy = 'date', $number = -
 		$payments = new \Give_Payments_Query( $args );
 		$payments = $payments->get_payments();
 
+		/**
+		 * Query renewal payments based on parent payments.
+		 * Because the renewal does not have a payment mode stored
+		 * the results are not included in the payment query,
+		 * so we have to run a second query and combine results.
+		*/
+		$payments = array_merge(
+			$payments,
+			$this->getRenewalPayments( $startStr, $endStr, $gateway )
+		);
```

Comment:
I would imagine both queries should be sorted by `DATE DESC`, which case after merging these we would need to do a `usort()` to put them in order.